pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md

```diff
@@ -0,0 +1,98 @@
+
+When you create a campaign you will select one of these actions to begin the workflow. In most cases this initial step will be an email sent to your segments.
+
+![](send-email-delay.png)
+
+You will notice that when you add an email to a campaign you will be able to select a potential *delay* for when the email is delivered. If the action is attached to a decision's non-action initiated decision path, the delay becomes how long the contact has to take action before the campaign progresses down the non-action path. 
+
+![](send-email-delay-nonaction.png)
+
+After you have added an action you will more than likely place a decision on the campaign.
+
+### Decisions
+
+Decisions are actions which are initiated by the contact. These decisions can be either directly initiated or implied based on non-action. Samples of these decisions are downloading an asset, opening an email, submitting a form, or visiting a landing page.
+
+Decisions are taken in response to an action and as such a decision has two outcomes.
+
+![](campaign-decisions.gif)
+
+These two options are demonstrated by the green and red decision points on the decision. Each path can then be handled by your campaign. This process is typically referred to as a **decision tree**.
+
+__It is important to note that a contact must already be part of the campaign in order for it to recognize the decision executed. Therefore, campaigns should never start with a decision unless you are manually managing the contacts assigned to it and the decision is expected to be executed at a later time.__ 
+
+#### Contact-initiated Decision Path (Green Points)
+
+Actions attached to the green point of a decision are considered contact-initiated points.
+
+The contact-initiated decision path is taken as a result of a contacts direct action such as opening an email or submitting a form. Connected actions will be executed (or scheduled if a delay is set) at the time the contact took the action. 
+
+#### Non-action Initiated Decision Path (Red Points)
+
+Actions attached to the red point of a decision are considered non-action points. This path is taken as a result of a contact NOT taking some direct action.
+
+Use an action's delay settings to define at what point the campaign should send the contact down this path.
+
+To trigger these events, see [executing campaign actions][executing-campaign-actions].
+
+#### Example
+
+To provide a simple example of a decision tree consider an email where the decision is to open an email. There are two outcomes, if the contact chooses to open the email then the green decision point connects to the next action to be taken in the campaign workflow. If, however, the contact does not open the email then you may desire a different action to be taken (e.g. a delay of 30 days then a second email sent).
+
```

Comment:
```suggestion
```

Reply:
This comment was on an older file that was merged. This file doesn't exist any longer.